src/pybind/mgr/dashboard/frontend/src/app/shared/components/notifications-sidebar/notifications-sidebar.component.html

```diff
@@ -120,9 +122,7 @@ <h6 class="card-title bold">{{ notification.title }}</h6>
   </div>
 </ng-template>
 
-<div class="card"
-     (clickOutside)="closeSidebar()"
-     [clickOutsideEnabled]="isSidebarOpened">
+<div class="card">
```

Comment:
Do we actually need to remove this functionality? I think it is a nice feature
`<div class="card"
     (clickOutside)="closeSidebar()"
     [clickOutsideEnabled]="isSidebarOpened">`